src/server/common/helpers/audit/auditor.js

```diff
@@ -1,15 +1,48 @@
-import { DevAuditor } from '~/src/server/common/helpers/audit/impl/dev-auditor'
-import { AwsAuditor } from '~/src/server/common/helpers/audit/impl/aws-auditor'
-import { config } from '~/src/config/config'
-
-function createAuditor(source) {
-  if (config.get('auditEnabled')) {
-    // Live auditor for production use
-    return new AwsAuditor(source)
-  } else {
-    // Local development & debug auditor
-    return new DevAuditor(source)
+import { noop } from 'lodash'
+
+import { config } from '~/src/config'
+import { DevAuditor } from '~/src/server/common/helpers/audit/dev-auditor'
+import { AwsAuditor } from '~/src/server/common/helpers/audit/aws-auditor'
+
+/**
+ * If audit is enabled, provide live AWS auditor for production use, if in development provide a local
+ * development and debug auditor
+ * @param options
+ * @returns {AwsAuditor|DevAuditor|{send: (...args: any[]) => void}}
+ */
+function createAuditor(options) {
+  if (options.audit.enabled) {
+    return new AwsAuditor(options)
+  }
+
+  if (options.isDevelopment) {
+    return new DevAuditor(options)
+  }
+
+  return {
+    send: noop
+  }
+}
+
+const auditor = {
+  name: 'auditor',
+  register: async (
+    server,
+    {
+      audit,
```

Comment:
where is `audit` from?

Reply:
Ah is that the input. I see

Reply:
Over [here](https://github.com/DEFRA/cdp-portal-frontend/pull/460/files#r1670901646)